src/components/ForgotPassword/forms/BottomButtonGroup.tsx

```diff
@@ -12,18 +12,20 @@ export type BottomButtonGroupProps = {
   disableBtn?: boolean;
   onresend?: () => void
   countdown?: number
+  backtoLogin?: string;
 };
 
 export const BottomButtonGroup: React.FC<BottomButtonGroupProps> = ({
-  continueButtonLabel = "Continue",
+  continueButtonLabel = "Confirm Email",
```

Comment:
Kindly revert this changes back to "Continue" , this props can be change when you reuse BottomButtonGroup (e.g <BottomButtonGroup continueButtonLabel="Confirm Email" />